examples/datepicker/datepicker.value.tsx

```diff
@@ -0,0 +1,54 @@
+import { type CodeData } from '~/components/code-demo';
+import { Datepicker } from '~/src';
+
+const code = `
+'use client';
+
+import { Datepicker } from 'flowbite-react';
+
+function Component() {
+  return (
+    <Datepicker
+      value={new Date(2022, 11, 25)} // 25th December 2022
+    />
+  );
+}
+`;
+
+const codeRSC = `
+import { Datepicker } from 'flowbite-react';
+
+function Component() {
+  return (
+    <Datepicker
+      value={new Date(2022, 11, 25);} // 25th December 2022
```

Comment:
extra comma here:
```tsx
value={new Date(2022, 11, 25);}
```

Reply:
also, do we need the comment there? since I can see its already outdated

Reply:
ohhhh man i am rocking it today so here is a work around for Datepicker **PLACEHOLDER** 

![Screenshot 2024-01-16 222752](https://github.com/themesberg/flowbite-react/assets/11619188/231d26c5-fd7a-404e-821f-56462dfe6103)

`value={formDefaultValues.start_date ? moment(new Date(formDefaultValues.start_date)).format('YYYY-MM-DD') : 'Please select start date'}`